.github/workflows/build.yml

```diff
@@ -9,26 +9,36 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
     name: Build and tests
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+        java: [ 8, 11, 15, 17 ]
```

Comment:
why 15 but not 18? would be useful to test against the latest version out there in additions to the LTSs.